.github/workflows/vsphere-e2e.yaml

```diff
@@ -84,3 +84,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
           VSPHERE_SERVER: ${{ secrets.VSPHERE_SERVER }}
           VSPHERE_DATASTORE: ${{ secrets.VSPHERE_DATASTORE }}
+          VSPHERE_DATACENTER: dc1
```

Comment:
Is this not a `secret` already?

Reply:
We don't need it to particularly be a secret and is only relevant for these builds so i added it here